requirements/runtime.txt

```diff
@@ -1 +1 @@
-Django>=2.2,<5.2
+Django>=2.2,<6.0
```

Comment:
```suggestion
Django>=2.2
```

Reply:
We do not want another
* #703

Reply:
> We do not want another
> 
> * [Django v5.0: django.utils.timezone.utc alias --> datetime.timezone.utc #703](https://github.com/celery/django-celery-beat/pull/703)

@cclauss you're comparing **major** version incompatibility, but <6.0 allows **minor** bump, which is usually safe.
I can't imagine django breaking changes in minor version.

Reply:
@cclauss ^ This problem could have been fixed by the end user with the use of monkey patching. User had a choice.

Reply:
@ivan-klass Please re-read the full commit message.

Reply:
> @ivan-klass Please re-read the full commit message.

Ahh, right! Sorry, I was lost in the discussion. I was looking forward for 5.2 compatible release, but don't want to block in case 5.3 and 5.4 next time. I'm not that experience for a strict opinion here, I would refer to best practices - i.e. how well-known django-based libraries do.